gems/pending/Gemfile

```diff
@@ -33,7 +33,8 @@ gem "net-sftp",                "~>2.1.2",           :require => false
 gem "nokogiri",                "~>1.6.8",           :require => false
 gem "openscap",                "~>0.4.3",           :require => false
 gem "openshift_client",        "=1.2.0",            :require => false
-gem "ovirt",                   "~>0.11.0",          :require => false
+gem "ovirt",                   "~>0.13.0",          :require => false
+gem "parallel",                "~>1.9",             :require => false # For OvirtInventory
```

Comment:
I'm pretty sure we already have parallel (via paralell_tests), so this comment is not completely accurate, but then again it would be nice to know why it's used.

@blomquisg This was part of my reason for asking about separating more providers.